src/artm/utility/progress_printer.cc.in

```diff
@@ -48,7 +42,7 @@ void ProgressPrinter::Set(size_t pos) noexcept {
   int pct = static_cast<int>((pos_ * 100) / max_);
   if (pct != prevpct) {
     // ANSI code \033[<N>D moves cursor to N positions backwards
-    fprintf(stderr, " %3d%%\033[5D", pct);
+    fprintf(stderr, " %3d%%\b\b\b\b\b", pct);
```

Comment:
I see, after all `\b` is a backspace - e.g. remove previous symbol. Strangely, this is not how it works on windows. From what I've tested it actually moves the cursor back without erasing characters.